### DISCUSSION TITLE
Wezterm is starting maximized even though it should fit my screen

### DISCUSSION THREAD

**Discussion by FluxZA** [wezterm/wezterm#2594] [Q&A] [answered]
I am asking this as a discussion question because I am not sure if it is a wezterm issue or a quirk in Fedora 36 Gnome with Wayland.

I adjusted the initial rows/cols values in my config to match the size I wanted. I found the size using the `wezterm cli list` command after manually dragging the window to the size I want. However, any value for rows above 34 results in wezterm starting maximized. If I restore the window, it is then the correct size as set in my config.

I searched for anything like this with Fedora, but didn't find anything. When looking at wezterm  issues/discussions, I found an issue mentioning that this behaviour is expected if the size is larger than your screen, but the size I have set fits on my screen (I also checked, it fits on my 2 secondary monitors as well).

Is this something I should create an issue for?

**Comment by wez** [comment, marked as answer]
It sounds like the windowing environment might decide that the size is close enough to maximized and tells wezterm to maximize

**Reply by FluxZA** [reply]
Okay, thanks for the info.
